util/config_map.py

```diff
@@ -55,28 +55,32 @@
 '''
 
 import ast
+from collections import OrderedDict
 import collections.abc
 import pprint
 import sys
+from typing import Union
 
 from absl import logging
-from collections import OrderedDict
-from typing import Union
+import file_util
 
 
 class ConfigMap:
-    '''Class to store config mapping of named parameters to values as a dictionary.'''
-
-    def __init__(self,
-                 config_dict: dict = None,
-                 filename: str = None,
-                 config_string: str = None):
-        '''Create a Config Map object.
-        Args:
-          config_dict: dictionary with key:values to be loaded into the config map.
-          filename: override the dictionary with key:values from the file.
-          config_string: string of dictionary parameters to override key:values.
-        '''
+    """Class to store config mapping of named parameters to values as a dictionary."""
```

Comment:
Looks like a formatting change (indentation + ''' -> """). Is this per Google Style guide recommendation?

Reply:
This was an automated change running `pylint` when yapf was deprecated in google3.

`yapf --style google` allows both and I have switched back to using it for more recent code.

Reply:
lgtm